pages/linux/systemd-run.md

```diff
@@ -0,0 +1,28 @@
+# systemd-run
+
+> Run programs in transient scope units, service units, or path-, socket-, or timer-triggered service units.
+> More information: <https://www.freedesktop.org/software/systemd/man/systemd-run.html>.
+
+- Run a program as root:
+
+`systemd-run {{executable}}`
+
+- Run a program in the user service, keeping the working (d)irectory and waiting until it exits:
```

Comment:
```suggestion
- Run a program in the user service, keeping the working [d]irectory and waiting until it exits:
```
`[]` is what we use to highlight longer options.

Reply:
Duh, not sure how did I miss this. All fixed.